Userland/Libraries/LibWeb/HTML/NavigableContainer.h

```diff
@@ -64,7 +64,7 @@ class NavigableContainer : public HTMLElement {
     // https://html.spec.whatwg.org/multipage/iframe-embed-object.html#navigate-an-iframe-or-frame
     void navigate_an_iframe_or_frame(URL::URL url, ReferrerPolicy::ReferrerPolicy referrer_policy, Optional<String> srcdoc_string = {});
 
-    WebIDL::ExceptionOr<void> create_new_child_navigable(JS::SafeFunction<void()> afterSessionHistoryUpdate = {});
+    WebIDL::ExceptionOr<void> create_new_child_navigable(JS::Handle<JS::HeapFunction<void()>> after_session_history_update = {});
```

Comment:
Why `Handle` and not `GCPtr`?

Reply:
See: https://github.com/LadybirdBrowser/ladybird/pull/1108/commits/3b9eb5dd4ff8bfaa67179367fa2e44999c3e5551

I'm not sure if that is safe? That commit does fix it up though by using HeapFunction through instead of HeapFunction->Function->HeapFunction

Reply:
Oh ha, me reading commits in chronological order and commenting :)